astropy/wcs/tests/extension/setup.py

```diff
@@ -26,17 +26,20 @@
     else:
         define_macros = []
 
-    try:
-        numpy_include = np.get_include()
-    except AttributeError:
-        numpy_include = np.get_numpy_include()
+    numpy_include = np.get_include()
+    wcs_include = wcs.get_include()
+    astropy_wcs_include = os.path.join(wcs.get_include(), 'astropy_wcs')
+    wcslib_include = os.path.join(wcs.get_include(), 'wcslib')
+    if not os.path.exists(os.path.join(wcslib_include, 'wcsconfig.h')):
```

Comment:
Huh... Why no one else complained about this? Maybe @nden wants to have a look here.

If this is fixing a real installation bug, maybe it needs a change log too?

Reply:
Oh... I didn't notice this is *inside* the tests directory. No need for change log then. Sorry for the noise!